File: hastur-client/tests/plugins/extended_hastur_plugin.pl
#!/usr/bin/env perl

use strict;
use warnings;
use JSON;

my $report = {
  status  => "OK",
  exit    => 0,
  message => "perl extended plugin works fine",
  stats   => [
    { runtime => 0.0, units => "s" }
  ],
  tags => ["version_0.1", "perl", "hastur"]
};

print encode_json($report), "\n";
exit 0

File: tests/plugins/extended_hastur_plugin.rb
#!/usr/bin/env ruby

require 'multi_json'

report = {
  :status  => "OK",
  :exit    => 0,
  :message => "ruby extended plugin works fine",
  :stats   => [
    { :runtime => 0.0, :units => "s" },
  ],
  :tags => ["version_0.1", "ruby", "hastur"],
}

puts MultiJson.encode(report)
exit 0

